src/System.Private.CoreLib/shared/System/Threading/ThreadInterruptedException.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-//
 /*=============================================================================
```

Comment:
Pleaes delete the comments on each of these files that are marked with `=========`. They are not useful.

Reply:
Actually, there is some value. I suggest you change them to `<summary>` on the class.